scripts/battlefields/Mine_Shaft_2716/return_to_the_depths.lua

```diff
@@ -0,0 +1,73 @@
+-----------------------------------
+-- Return to the Depths
+-- Mine Shaft #2716 quest battlefield
+-----------------------------------
+local mineshaftID = zones[xi.zone.MINE_SHAFT_2716]
+-----------------------------------
+
+local content = BattlefieldQuest:new({
+    zoneId        = xi.zone.MINE_SHAFT_2716,
+    battlefieldId = xi.battlefield.id.RETURN_TO_THE_DEPTHS,
+    canLoseExp    = false,
+    maxPlayers    = 6,
+    timeLimit     = utils.minutes(30),
+    index         = 1,
+    entryNpc      = '_0d0',
+    exitNpcs      = { '_0d1', '_0d2', '_0d3' },
+
+    questArea     = xi.questLog.BASTOK,
+    quest         = xi.quest.id.bastok.RETURN_TO_THE_DEPTHS,
+
+    requiredVar   = 'Quest[1][78]prog',
+    requiredValue = 9,
+})
+
+function content:onEventFinishWin(player, csid, option, npc)
```

Comment:
This function can be most likely removed if you set the gil reward in the constructor above, and then capture the 32001 event in the quest script

Reply:
I'm not seeing gil listed [here](https://github.com/LandSandBoat/lsb-wiki/blob/main/Battlefield-Framework-Documentation.md#battlefield-parameters) either